templates/include/roles.jsonnet

```diff
@@ -8,6 +8,19 @@
         "name": "enmasse-address-admin"
       },
       "rules": [
+        {
+          "apiGroups": [
+            ""
+          ],
+          "resources": [
+            "routes"
```

Comment:
why is this needed? how does it know the route name to look up?  its not necessarily named 'messaging' ,because it can be configured when creating address space.

Reply:
The route is used to adjust the online help doc to have the actual hostname in it. Not having permission is a regression for that feature. If the route is named something else, this adjustment won't happen (just as it doesn't happen without the correct permission). I've added an env var to pass through the route name to lookup. However at the moment, I think fixing the permissions to allow the existing feature to work again is valuable as for those users who don't override the route name, it make it easy to find (especially if they don't have access to the actual route object in openshift). Expanding to cover the more advanced case or indeed revising the mechanism entirely can be done as a separate PR.

Reply:
Agreed this is sufficient for now.